gnome-extra/cinnamon/cinnamon-5.0.3.ebuild

```diff
@@ -128,16 +128,15 @@ src_configure() {
 		$(meson_use gstreamer build_recorder)
 		$(meson_use gtk-doc docs)
 		-Ddisable_networkmanager=$(usex networkmanager false true)
+		-Dpy3modules_dir="$(python_get_sitedir)"
 	)
 	meson_src_configure
 }
 
 src_install() {
-	python_domodule files/usr/lib/python3/dist-packages/cinnamon
-	rm -rf files/usr/lib/python3 || die
-
 	meson_src_install
 
+	python_optimize "${ED}$(python_get_sitedir)"
```

Comment:
Not sure this is semantically correct, but it works correctly on my non-prefix system.

Reply:
`${ED}$(python_get_sitedir): concatenates two paths containing EPREFIX on line 139`

Presuming `python_get_sitedir` contains `EPREFIX` because it's running in `src_install`(?), should this be `${D}$(python_get_sitedir)`? Or just plain old `$(python_get_sitedir)`?